src/main/java/uk/gov/justice/digital/delius/controller/api/OffenderDeltaController.java

```diff
@@ -29,19 +31,21 @@ public OffenderDeltaController(OffenderDeltaService offenderDeltaService) {
     }
 
     @RequestMapping(value = "/offenderDeltaIds", method = RequestMethod.GET)
-    public ResponseEntity<List<OffenderDelta>> getOffenderDeltas() {
+    @JwtValidation
```

Comment:
Unrelated; but missing meaning this was always unprotected and using the "wrong" user (which had no behaviour impact)